lib/triplex.ex

```diff
@@ -138,12 +151,23 @@ defmodule Triplex do
     if reserved_tenant?(tenant) do
       {:error, reserved_message(tenant)}
     else
-      sql = "DROP SCHEMA \"#{to_prefix(tenant)}\" CASCADE"
+      adapter = repo.__adapter__
+      sql = case adapter do
+        Ecto.Adapters.MySQL -> "DROP DATABASE #{to_prefix(tenant)}"
+        _ -> "DROP SCHEMA \"#{to_prefix(tenant)}\" CASCADE"
+      end
       case SQL.query(repo, sql, []) do
+        {:ok, _} when adapter == Ecto.Adapters.MySQL ->
+          case SQL.query(repo, "DELETE FROM #{Triplex.config().tenant_table} WHERE NAME = ?", [tenant]) do
+            {:error, msg} -> {:error, msg}
+            _ -> {:ok, tenant}
+          end
```

Comment:
We could refactor this nested `case`s to use a `with`

Reply:
Done